src/constants.js

```diff
@@ -58,7 +58,8 @@ export const overViewNames = [
   "Client",
   "StartDate",
   "EstimatedEndDate",
-  "EndDate"
+  "EndDate",
+  "parentName"
```

Comment:
That name is not complex with "EndDate" etc.. camelCase is not used.

Reply:
Corrected name but left '@media'.